static/js/message_events.js

```diff
@@ -484,14 +483,13 @@ export function update_messages(events) {
         // edit.  Doing so could save significant work, since most
         // topic edits will not match the current topic narrow in
         // large organizations.
-        if (!changed_narrow && message_lists.current === message_list.narrowed) {
-            message_list.narrowed.update_muting_and_rerender();
+        if (!changed_narrow && message_lists.current.is_current_view()) {
+            message_lists.current.update_muting_and_rerender();
         }
     } else {
         // If the content of the message was edited, we do a special animation.
-        message_lists.current.view.rerender_messages(msgs_to_rerender, message_content_edited);
-        if (message_lists.current === message_list.narrowed) {
-            message_lists.home.view.rerender_messages(msgs_to_rerender);
+        for (const list of message_lists.all_rendered_message_lists()) {
+            list.view.rerender_messages(msgs_to_rerender, message_content_edited);
```

Comment:
We previously did not pass `message_content_edited` to the home message list version of this. I think it was likely  a bug that we didn't do so, but wanted to flag this -- I might split out a fix to this as a prep commit.

Reply:
This as a separate commit doesn't makes sense because here:
```js
        message_lists.current.view.rerender_messages(msgs_to_rerender, any_message_content_edited);
        if (message_lists.current === message_list.narrowed) {
            message_lists.home.view.rerender_messages(msgs_to_rerender);
        }
```

`message_lists.home` is not visible to the user. So, passing `any_message_content_edited` to it inside the if statement will have no effect.